src/python/convert_numpy_features_to_protos.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from pathlib import Path
+import argparse
+import protos_io
+
+import protos.localization_protos_pb2 as loc_protos
+
+
+def convert_to_protos(features, feature_type):
+    """Converts the numpy nd array to the features protos.
+
+    Args:
+        features (numpy.ndarray): feature vectors to be converted.
+                                Expected size NxD, where N is
+                                number of features and D is number
+                                of dimensions.
+    Returns:
+        [image_sequence_localizer.Feature]: list of feature protos.
+                                            For details check
+                                            localization_protos.proto file.
+    """
+    protos = []
+    assert len(features.shape) == 2
+    for feature in features:
+        feature_proto = loc_protos.Feature()
+        feature_proto.size = feature.shape[0]
+        feature_proto.type = feature_type
+        feature_proto.values.extend(feature)
+        protos.append(feature_proto)
+    return protos
+
+
+def save_protos_to_files(folder, protos, prefix):
+    if folder.exists():
+        print("WARNING: the folder exists. Potentially overwritting the files.")
+    else:
+        folder.mkdir()
+    for idx, proto in enumerate(protos):
+        feature_idx = "{0:07d}".format(idx)
+        proto_file = prefix + "_" + feature_idx + "." + proto.type + ".Feature.pb"
```

Comment:
nit: an alternative syntax for this would be:
```python
proto_file = "{prefix}_{feature_idx}.{proto_type}.Feature.pb".format(
  prefix=prefix,
  feature_idx=feature_idx,
  proto_type=proto.type)
```
I personally like it more than concatenating strings but it's not too important.

Reply:
Cool, that looks much better, thanks. Will change that.